.github/workflows/pages.yaml

```diff
@@ -63,8 +71,24 @@ jobs:
       - name: Get the list of tagged revisions (for cache)
         id: get-revisions
         run: |
-          echo revisions_hash=$(cd support/OSCAL; git tag | grep -E '^v\d+\.\d+\.\d+$' | sha256sum) >> $GITHUB_OUTPUT
+          echo revisions_hash=$(make --silent list-tags | sha256sum) >> $GITHUB_OUTPUT
         shell: bash
+      - name: Get the list of prototype branches (for cache)
+        id: get-prototype-branches
+        run: |
+          {
+            # Wildcards after a stable tag prefix like v* for v1.0.0 will not work, so generate outputs for content and data paths for next step
+            branches="$(make --silent list-prototype-branches PROTOTYPE_BRANCHES_PREFIX=$PREFIX)"
+            echo branches="$branches"
+            [[ -n "${branches-}" ]] && echo content_paths="$(printf "site/content/models/%s " $branches)"
+            [[ -n "${branches-}" ]] && echo data_paths="$(printf "site/content/data/%s " $branches)"
+            # Be forewarned, printf -- has a special meaning so we must encode the first - as \055 in -- to avoid errors
+            # Explanation: https://unix.stackexchange.com/a/658388
+            [[ -n "${branches-}" ]] && echo linkcheck_exclude_path_arg=$(printf "\055-exclude-path='site/public/models/%s' "  $branches)
```

Comment:
NB: That sounds like it was painful to debug, nice catch

Reply:
Well, I got through it! 😅